Gaijinizer patch/Map220.txt

```diff
@@ -330,9 +366,10 @@ I thought what would happen at one time ...#MTLed
 ニーナは、人間界と魔界に対して
 定期的に二種類の番組を放送している。
 > CONTEXT: Map220/events/1/pages/0/196/Dialogue
-After that escape play ......... Nina is against the human
-realm and the Demon Realm. We broadcast two kinds of
-programs regularly.#MTLed
+After that escape drama.........
+
+Nina has been regularly broadcasting two programs
+for the Human World and the Demon World.#Strate
 > END STRING
```

Comment:
Some rewording and rephrasing here:
```
Ever since the story of my escape came to a
close, Nina began regularly broadcasting two
programs for the Human World and the Demon World.
```